<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>LoadingDialog</class>
    <widget class="QDialog" name="LoadingDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>450</width>
                <height>316</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>450</width>
                <height>316</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>450</width>
                <height>316</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Loading...</string>
        </property>
        <property name="styleSheet">
            <string>
                QDialog {
                background-color: #ffffff; /* Set the background color to white */
                border-radius: 10px;
                border: 1px solid #ccc;
                }
            </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="loadingLabel">
                    <property name="styleSheet">
                        <string notr="true">font-size: 18pt; font-weight: bold; color: #555;</string>
                    </property>
                    <property name="text">
                        <string>Loading, please wait...</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <spacer name="horizontalSpacerLeft">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Expanding</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>0</width>
                                    <height>0</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QLabel" name="movieLabel">
                            <property name="minimumSize">
                                <size>
                                    <width>50</width>
                                    <height>50</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>50</width>
                                    <height>50</height>
                                </size>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacerRight">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Expanding</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>0</width>
                                    <height>0</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QListWidget" name="checkListWidget">
                    <property name="styleSheet">
                        <string notr="true">
                            QListWidget::item {
                            font-size: 14pt;
                            color: #333;
                            padding: 5px;
                            }
                            QListWidget::item::checked {
                            color: #2A9D8F;
                            text-decoration: line-through;
                            }
                        </string>
                    </property>
                    <item>
                        <property name="text">
                            <string>Reading sequence data</string>
                        </property>
                    </item>
                    <item>
                        <property name="text">
                            <string>Aligning sequences</string>
                        </property>
                    </item>
                    <item>
                        <property name="text">
                            <string>Constructing genetic trees</string>
                        </property>
                    </item>
                    <item>
                        <property name="text">
                            <string>Displaying results</string>
                        </property>
                    </item>
                    <item>
                        <property name="text">
                            <string>Saving results</string>
                        </property>
                    </item>
                </widget>
            </item>
            <item>
                <widget class="QProgressBar" name="progressBar">
                    <property name="styleSheet">
                        <string notr="true">
                            QProgressBar {
                            border: 1px solid #bbb;
                            border-radius: 5px;
                            background-color: #eee;
                            text-align: center;
                            }
                            QProgressBar::chunk {
                            background-color: #2A9D8F;
                            width: 20px;
                            }
                        </string>
                    </property>
                    <property name="minimum">
                        <number>0</number>
                    </property>
                    <property name="maximum">
                        <number>100</number>
                    </property>
                    <property name="value">
                        <number>0</number>
                    </property>
                    <property name="textVisible">
                        <bool>false</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>